gotv_transportation.md

```diff
@@ -0,0 +1,169 @@
+| coordinating_rider | Rider* | If set, then communications about an assigned Ride should be directed towards this Rider. It is an error if this value is set but is not associated with an element of `riders`
+
+Systems may reject creation of demands which would result in a rider being associated with multiple demands.
+
+### Rider Fields
+
+| Name | Type | Description
+|------|--------|------
+| id | string | The unique ID for this Rider
+| person | Person* | If set, then this Rider is associated with a known [Person](people.html).
+| assistant | Person* | If set, then this Rider is not the principal source of the associated Demand, but requires transport along with the one who is in order to help that person through the voting or transportation process. Examples might include healthcare or nursing aides or translators.
+| minor | int* | If set, the Rider is a minor accompanying another Rider in the group, and has the age indicated. As this Rider is a minor, the system has no expectation or desire that they be personally identified as an instance of `Person`.
+| restrictions | RiderRestriction[] | Any Rider restrictions which should be considered when creating an Assignment for this Rider
+
+Note that of the fields `person`, `assistant`, and `minor`, exactly one must be set in each instance.
+
+### RiderRestriction Fields
+
+| Name | Type | Description
+|------|--------|------
+| key | string | The well-known key to the system identifying the nature of this restriction.
+| value | object | The value expressing the specifics of the restriction, as appropriate for its type.
+
+Restrictions are loosely typed and specific to the system responsible for making Assignments. This specification may evolve to provide some basic suggested restrictions which could be broadly useful, but there would be no requirement that implementations express semantically equivalent restrictions in the same way. The values for `key` are intended to be globally unique, and should be namespaced in such a way that they will not collide with restrictions from other providers. For instance, good keys might take the form of `osdi:rider_restriction_language` or `acme:requires_wheelchair_ramp`, whereas simply `requires_wheelchair_ramp` should be avoided as likely to collide.
+
+#### Standard Values
+
+We can imagine the following starting point of restrictions for use by systems that find them useful. In each case below, we provide a type definition for the object passed as the `value` for the restriction, as well as provide a suggested value for `key`.
+
+##### Language
+
+Keyed as `osdi:rider_restriction_language`:
+
+| Name | Type | Description
+|------|--------|------
+| iana_codes | string[] | Language codes, as defined by [RFC 5646](https://tools.ietf.org/html/rfc5646), that the associated rider can speak. Examples would include `en/US` or `es/419`.
+
+##### Accessibility
+
+Keyed as `osdi:rider_restriction_accessibility`
```

Comment:
a subset of these riders might also need a caretaker to travel with them. 
Consider adding something like osdi:rider_restriction_caretaker` - this would indicate that this rider requires an assistant or caretaker to ride with them (thus taking another seat in the vehicle).